### ISSUE TITLE
Why do the input keypoints have a large spatial deviation from the generated smpl

### ISSUE THREAD

**Issue by keylLi** [wangsen1312/joints2smpl#10]
Why do the input keypoints have a large spatial deviation from the generated smpl body 。

![微信图片_20240314175350](https://github.com/wangsen1312/joints2smpl/assets/46370902/c2111067-e3ea-4d85-862c-e1c593d5d12b)

**Comment by wangsen1312** [comment]
How you render the SMPL model to the image?

**Comment by keylLi** [comment]
> How you render the SMPL model to the image?

@wangsen1312 I was using open3d to draw the vertices and triangles from the smplmodel output as well as the raw joints from the inputs，the code is : vis_mesh_pcd_o3d(np.array(mesh_p.as_open3d.vertices), np.array(mesh_p.as_open3d.triangles),
 joints3d.squeeze().detach().cpu().numpy()). Then, then  image above shows up, and it's clear that there's a rotation and a translation between them.
Do you know what is causing this? Thanks for the answer.
